config/async/100-async-producer.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       - image: ko://knative.dev/async-component/cmd/producer
         env:
         - name: REDIS_ADDRESS
-          value: "rediss://user:userpassword@asdf1234.databases.appdomain.cloud:30285"
+          value: "redis.redis.svc.cluster.local:6379"
```

Comment:
Can this be added dynamically? See ytt for a tool to add / modify YAML files if this value can be determined automatically